clients/fides-js/src/lib/consent.tsx

```diff
@@ -20,6 +19,16 @@ export const initOverlay = async ({
   debugLog(options.debug, "Initializing Fides consent overlays...");
 
   async function renderFidesOverlay(): Promise<void> {
+    // Check if there are any notices within the experience that do not have a user preference
+    const noticesWithNoUserPreferenceExist: boolean = Boolean(
+      experience?.privacy_notices?.some(
+        (notice) => notice.current_preference == null
+      )
+    );
+    const modalLinkEl = document.getElementById(FIDES_MODAL_LINK);
+    if (!(noticesWithNoUserPreferenceExist || modalLinkEl)) {
```

Comment:
this is a little confusing to me—why do we skip rendering the overlay if there's no modal link?

Reply:
fixed- _only_ if we don't have new notices and modal link does not exist, we shouldn't render overlay